tools/topology/sof-hda-generic-idisp.m4

```diff
@@ -0,0 +1,97 @@
+#
+# Topology for generic Apollolake UP^2 with pcm512x codec and HDMI.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
```

Comment:
@plbossart And we dont need to include ssp.m4

Reply:
ok